dist/src/test/java/io/camunda/application/CamundaDockerIT.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
+import org.apache.hc.client5.http.classic.methods.HttpGet;
+import org.apache.hc.client5.http.impl.classic.CloseableHttpResponse;
+import org.apache.hc.client5.http.impl.classic.HttpClients;
+import org.apache.hc.core5.http.io.entity.EntityUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+
+@EnabledIfSystemProperty(named = "camunda.docker.test.enabled", matches = "true")
+public class CamundaDockerIT {
```

Comment:
❓ Root QA module is now available on main, do you want to move it now?

Reply:
yes, I have the same idea. I am planning to do it in a a new pull request.
I was planning to create a new module under QA module, dedicated to this kind of tests, with the minimum possible of dependencies: testContainers, jUnit5, apache http5...
we don't really need any dependency to the applicative code, and it should be easy to build and run tests. WDYT @Zelldon ?

Reply:
Separate PR is fine by me. Extra module not sure whether this is necessary. I think you can just put it into the integration-tests module in a different package.

Reply:
This test is not run within integration tests suite, but only on newly built camunda docker image (conditional on `camunda.docker.test.enabled` property). When we need to run this test only, we don't really need to do a full project build or import dozens of applicative snapshot artifacts that we don't anyway use in the test.